lib/lknvball_web/schema/schema.ex

```diff
@@ -42,27 +42,42 @@ defmodule LknvballWeb.Schema do
       %Lknvball.Events.Event{}, _ ->
```

Comment:
So this needs to be refactored into a function, honestly I probably need to figure out how to macro it. But for now until the application grows to 15 + models or so lets leave it as is!

Reply:
You talking about the node resolver specifically?

I can throw a TODO in there.